src/dsp_tools/commands/excel2json/models/input_error_lists.py

```diff
@@ -32,7 +33,27 @@ def execute_error_protocol(self) -> str:
 
 
 @dataclass
-class ListSheetProblem:
+class CollectedSheetProblems:
+    problems: list[SheetProblem]
+
+    def execute_error_protocol(self) -> str:
+        raise NotImplementedError
+
+    def _sort_by_excel(self) -> dict[str, SheetProblem]:
+        raise NotImplementedError
+
+
+@dataclass
+class SheetProblem(Protocol):
```

Comment:
I would prefer defining this Protocol before its first usage. (It's used on line 37)